src/compute-client/src/protocol/command.rs

```diff
@@ -171,11 +171,12 @@ pub enum ComputeCommand<T = mz_repr::Timestamp> {
         frontier: Antichain<T>,
     },
 
-    /// `Peek` instructs the replica to perform a peek at an index.
+    /// `Peek` instructs the replica to perform a peek on a collection: either an index or a
+    /// Persist-backed collection.
     ///
     /// The [`Peek`] description must have the following properties:
     ///
-    ///   * The target index has previously been created by a corresponding `CreateDataflow`
+    ///   * If targeting an index, it has previously been created by a corresponding `CreateDataflow`
     ///     command.
```

Comment:
Is it also required that a targeted persist collection has been created in Persist before?